docs/utils.md

```diff
@@ -46,7 +46,7 @@ utils.verifyPassword(userRecord, "#1 CloudSafe!"); // true if this is their pass
 
 Returns a promise that will contain the user recored on a successful resolve.
 
-## `{that}.unlockUser(username, password)
+### `{that}.unlockUser(username, password)`
 
 * Takes a username and password as strings.
 * Returns the `userData` record if the password is correct, otherwise an `isError` Object.
```

Comment:
This is not correct.  It returns a promise.

Reply:
I think this should be matched up with the preferred format now.

Reply:
It looks good, but I would change the "promise" in the returns block to `fluid.promise`, linked to [the docs](https://docs.fluidproject.org/infusion/development/PromisesAPI.html), as the contract is different than what other people might think of as promises.

Reply:
Sorry for not pointing out the distinction in my earlier comment, I realise I literally said "promise" myself.